ansible/roles/passwords/templates/k3s-token.auto.tfvars.json.j2

```diff
@@ -0,0 +1,3 @@
+{
+    "k3s_token": "{{ slurm_appliance_secrets.k3s_token }}"
```

Comment:
making it idempotent means that they will still be mismatched on the initial run because ansible hasn't picked up the templated secrets yet because the role is still running, but they will be updated to be the same on subsequent runs. Added a workaround to fix this